cypress-all.json

```diff
@@ -0,0 +1,5 @@
+{
+  "baseUrl": "https://localhost:8642",
+  "fixturesFolder": false,
+  "testFiles": "all/**/*.spec.ts"
+}
```

Comment:
I think rather than having a separate `all` suite, we want to run these tests in both the `basic` and `core` tiers. That way tests here can ensure that shared functionality does actually work within both tiers.